src/Indice.Features.Cases.AspNetCore/Interfaces/ILookupService.cs

```diff
@@ -14,6 +14,6 @@ public interface ILookupService
         /// <param name="lookupName">The lookup name.</param>
         /// <param name="searchValues">Any search values to filter the results.</param>
         /// <returns></returns>
-        Task<ResultSet<LookupItem>> Get(string lookupName, string searchValues = null);
+        Task<ResultSet<LookupItem>> Get(string lookupName, SearchValues searchValues = null);
```

Comment:
the type of searchValues parameter should be "client agnostic".

Reply:
Firstly, we should use ListOptions and, secondly, we should use something like `{'name':'...', 'value':'...'}`.